packages/astro/package.json

```diff
@@ -136,7 +136,7 @@
     "strip-ansi": "^7.0.1",
     "supports-esm": "^1.0.0",
     "tsconfig-resolver": "^3.0.1",
-    "vite": "^3.0.0",
+    "vite": "^3.0.2",
```

Comment:
I would consider this Vite making breaking changes behind patches, and not following semver. Can we pin to v3.0.2 for now until their v3 matures a bit?

Reply:
Done

Reply:
Sorry for this. FWIW, we counted it as regression as we didn't intend to change the way `noExternal/external` works. Given the current discussion in https://github.com/vitejs/vite/issues/9275, I think that external will end up working as it did in Vite 2 and also count as a regression fix. 
But I totally understand fixing the version until this is resolved 👍🏼